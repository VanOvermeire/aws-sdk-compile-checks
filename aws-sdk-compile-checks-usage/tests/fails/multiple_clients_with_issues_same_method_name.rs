use aws_sdk_compile_checks_macro::required_props;

#[required_props]
async fn evidently(evidently_client: aws_sdk_evidently::Client, sagemaker_client: aws_sdk_sagemaker::Client) {
    let _ = evidently_client.create_project()
        .send()
        .await;
    let _ = sagemaker_client.create_project()
        .send()
        .await;
}

fn main() {}
