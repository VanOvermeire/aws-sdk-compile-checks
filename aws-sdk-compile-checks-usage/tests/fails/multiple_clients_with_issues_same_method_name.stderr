error: method `create_project` (from sagemaker) is missing required argument(s): `project_name`, `service_catalog_provisioning_details`
 --> tests/fails/multiple_clients_with_issues_same_method_name.rs:8:30
  |
8 |     let _ = sagemaker_client.create_project()
  |                              ^^^^^^^^^^^^^^

error: method `create_project` (from evidently) is missing required argument(s): `name`
 --> tests/fails/multiple_clients_with_issues_same_method_name.rs:5:30
  |
5 |     let _ = evidently_client.create_project()
  |                              ^^^^^^^^^^^^^^
